docs/how-to/how-to-profile-memory.md

```diff
@@ -5,7 +5,7 @@ This crate provides an easy approach to dump memory profiling info.
 ## Prerequisites
 ### jemalloc
```

Comment:
If you don't mind, you can add a section for installing the correct version of jeprof. I wrote a doc for it but haven't submitted a PR.
https://github.com/GreptimeTeam/greptimedb/blob/build/add-jeprof/docs/how-to/how-to-profile-memory.md

Reply:
of course will add it now, and add you to co-author

Reply:
Done